pluggy/pluginmanager.py

```diff
@@ -0,0 +1,293 @@
+from . import _tracing
```

Comment:
Do we want to name our modules `twonames` or `two_names`? I ask because we have `parse_funcs` and now `pluginmanager`.

FWIW I prefer `two_names` as I find it easier to read. Either way we should choose a convention.

Reply:
thamks for that note, i#l ltry to incooperate it today

Reply:
What about just `manager.py`?